net-snmp/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+(
+    . /etc/sysconfig/snmpd
+    if test "$WAIT_FOR_CONFIGURATION" == 1; then
+        systemd-notify --ready
+        exec sleep 9999d
```

Comment:
sure.  I'd like to drop this hack though.  I didn't manage to find a way to create an user using the container.  The issue is that I cannot use "net-snmp-create-v3-user" when the daemon is running (it is not possible it seems), so I cannot "runc exec " into the container to create an user since there is no container running.  This hack is to set an environment variable so that the container just hangs and it is possible to "runc exec" into it and run commands.

We should probably find a way to run oneshot containers here